test/unit/pr_checker.test.js

```diff
@@ -193,6 +193,42 @@ describe('PRChecker', () => {
       assert(!status);
       assert.deepStrictEqual(logger.logs, expectedLogs);
     });
+
+    it('should skip wait check for Code & Learn PR', () => {
+      const logger = new TestLogger();
+
+      const expectedLogs = {
+        warn: [],
+        info: [['This PR was is Code & Learn and doesn\'t have a time limit.']],
+        error: [],
+        trace: []
+      };
+
+      const now = new Date('2017-10-28T13:00:41.682Z');
```

Comment:
@Tiriel is there a need for this.
Nit: Just pass `new Date()` to the `checker.checkPRWait`